views/default/panels/user/switch.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+use yii\bootstrap\ActiveForm;
+use yii\bootstrap\Html;
+use yii\grid\GridView;
+
+/* @var $this \yii\web\View */
+/* @var $panel yii\debug\panels\UserPanel */
+
+if ($panel->canSwitchUser()) {
+    echo '<h2>Switch user</h2>';
+?>
+<div class="row">
+    <div class="col-sm-7">
+        <?php $formSet = ActiveForm::begin([
+            'action' => \yii\helpers\Url::to(['user/set-identity']),
+            'layout' => 'horizontal',
+            'options' => [
+                'style' => ($panel->canSearchUsers() ? 'display:none' : '')
+            ]
+        ]);
+        echo $formSet->field(
+            $panel->userSwitch->getUser()->identity,
+            'id', ['options' => ['class' => '']])->textInput(['id' => 'user_id', 'name' => 'user_id'])
```

Comment:
This line causes error in case Indentity isn't an AR. Try using toolbar in basic application template.